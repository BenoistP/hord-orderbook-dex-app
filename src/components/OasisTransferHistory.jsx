import React, { PureComponent } from 'react';
import { PropTypes } from 'prop-types';
// import ImmutablePropTypes from 'react-immutable-proptypes';

import styles from './OasisTransferHistory.scss';

const propTypes = PropTypes && {};
const defaultProps = {};

class OasisTransferHistory extends PureComponent {
  render() {
    return (
      <div className={styles.base}>
      </div>
    );
  }
}

OasisTransferHistory.displayName = 'OasisTransferHistory';
OasisTransferHistory.propTypes = propTypes;
OasisTransferHistory.defaultProps = defaultProps;
export default OasisTransferHistory;
