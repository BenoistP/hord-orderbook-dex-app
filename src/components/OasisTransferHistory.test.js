/* global shallow describe it expect */
/* eslint-disable import/first */
import React from 'react';
import { shallow } from 'enzyme';
import OasisTransferHistory from './OasisTransferHistory';


describe('(Component) OasisTransferHistory', () => {
  it('should render', () => {
    const props = {
      children: (
        <span>test</span>
      )
    };
    const wrapper = shallow(
      <OasisTransferHistory {...props}/>
    );

    expect(wrapper).toMatchSnapshot();
  });
});
