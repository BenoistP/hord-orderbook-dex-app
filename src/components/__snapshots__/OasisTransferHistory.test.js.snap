// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`(Component) OasisTransferHistory should render 1`] = `<div />`;
